Mathlib/CategoryTheory/Shift/Opposite.lean

```diff
@@ -110,4 +112,67 @@ lemma oppositeShiftFunctorAdd'_hom_app :
   subst h
   simp only [shiftFunctorAdd'_eq_shiftFunctorAdd, oppositeShiftFunctorAdd_hom_app]
 
+end
+
+variable {C D : Type*} [Category C] [Category D] (F : C ⥤ D) (A : Type*) [AddMonoid A]
+  [HasShift C A] [HasShift D A]
+
+namespace CommShift
+
+open Functor
+
+/--
+Given a `CommShift` structure on `F`, this is the corresponding `CommShift` structure on
+`F.op` (for the naive shifts on the opposite categories).
+-/
+@[simps]
+noncomputable def commShiftOp [CommShift F A] :
```

Comment:
In terms of namespaces, this should be `CategoryTheory.Functor.commShiftOp` (and the one below `commShiftUnop`).

Reply:
Changed.